spec/factories/accounting_transactions.rb

```diff
@@ -1,13 +1,20 @@
 FactoryGirl.define do
-  factory :accounting_transaction, class: 'Accounting::Transaction' do
-    sequence(:qb_id)
-    qb_transaction_type { Accounting::Transaction::QB_TRANSACTION_TYPES.sample }
-    quickbooks_data { {} }
-  end
-
-  factory :journal_entry_transaction, class: 'Accounting::Transaction' do
+  factory :accounting_transaction, class: 'Accounting::Transaction', aliases: [:journal_entry_transaction] do
     sequence(:qb_id)
+    # All Madeline-related transactions are Journal Entries now
+    # qb_transaction_type { Accounting::Transaction::QB_TRANSACTION_TYPES.sample }
     qb_transaction_type 'JournalEntry'
     quickbooks_data { {} }
+    loan_transaction_type %w(Disbursement Repayment).sample
```

Comment:
These values are not capitalized in the transaction model. These values should be consistent with the model.